desktop/apps/article/client/index.coffee

```diff
@@ -113,26 +116,23 @@ module.exports = class ArticleIndexView extends Backbone.View
 
   setupPromotedContent: =>
     if @article.get('channel_id') is sd.PC_ARTSY_CHANNEL
-      new Partner(id: @article.get('partner_ids')?[0]).fetch
-        error: => $('.articles-promoted').hide()
-        success: (partner) ->
-          new Profile(id: partner.get('default_profile_id')).fetch
-            error: => $('.articles-promoted').hide()
-            success: (profile) ->
-              $('#articles-show').prepend promotedTemplate
-                src: profile.bestAvailableImage()
-                name: partner.get('name')
-                href: profile.href()
-                type: profile.profileType()
+      send = method: 'post', query: partnerQuery(@article.get('partner_ids')?[0])
+      metaphysics send
+      .then (data) =>
+        $('#articles-show').prepend promotedTemplate
+          src: data.partner.profile.image.cropped.url
+          name: data.partner.name
+          href: data.partner.profile.href
+          type: if data.partner.type is 'Gallery' then 'Gallery' else 'Institution'
```

Comment:
Fairs aren't currently modeled as Partners -- they are `FairOrganizer`'s so no need to worry about that. And no, we don't have promoted content for them.

I'd assume it'd be fine to derive the type from either Partner or Profile, but maybe it's a good question to ask #platform-humans. Basically I'm wondering how Partner `type` differs from Profile `owner_type`.

Looking at [Gravity's Partner model](https://github.com/artsy/gravity/blob/master/app/models/domain/partner.rb#L13), it looks like there are other types the API can return.